scripts/haproxy/s3haproxyconfig/s3haproxyconfig/s3_haproxy_config.py

```diff
@@ -56,7 +64,33 @@ def run(self):
     #Add backend information to haproxy.cfg file
     cfg_file = '/etc/haproxy/haproxy.cfg'
     target = open(cfg_file, "a+")
-    target.write('''# BackEnd roundrobin as balance algorithm
+    target.write('''#-------S3 Haproxy configuration start--------------------------------
+#---------------------------------------------------------------------
+# FrontEnd S3 Configuration
+#---------------------------------------------------------------------
+frontend main
+    # s3 server port
+    bind 0.0.0.0:80
+    #bind 0.0.0.0:443 ssl crt /etc/ssl/stx/stx.pem
+
+    option forwardfor
+    default_backend app-main
+
+
+
+    # s3 auth server port
+    bind 0.0.0.0:9080
+    #bind 0.0.0.0:9443 ssl crt /etc/ssl/stx/stx.pem
+
+
+
+    acl s3authbackendacl dst_port 9443
+    acl s3authbackendacl dst_port 9080
+    use_backend s3-auth if s3authbackendacl
+
+
+
+# BackEnd roundrobin as balance algorithm
```

Comment:
are we planning for handling changes of updating all instance ids of s3server when we have multi-node/multiple instance setup.

Reply:
Yes, that will be taken up in [EOS-16435](https://jts.seagate.com/browse/EOS-16435)